app/code/Magento/Checkout/Block/Cart/Totals.php

```diff
@@ -220,4 +238,14 @@ public function getQuote()
         }
         return $this->_quote;
     }
+
+    /**
+     * Get serialized checkout config.
+     *
+     * @return bool|string
+     */
+    public function getSerializedCheckoutConfig()
```

Comment:
Unfortunately, we couldn't approve adding a new public function to this API class.

Can you please inject cart config as a layout argument to this block and retrieve it in the template using magic getter (see https://github.com/magento/magento2/pull/18443/files for the example of implementation)

Reply:
Hi @sivaschenko! Thank you for your feedback. I've done the modifications to accomplish tha same result by using the layout.